openssl/src/cipher_ctx.rs

```diff
@@ -509,6 +509,14 @@ impl CipherCtxRef {
         Ok(())
     }
 
+    /// Set ctx flags.
+    #[corresponds(EVP_CIPHER_CTX_set_flags)]
+    pub fn set_flags(&mut self, flags: i32) {
```

Comment:
If this is going to be publicly exposed, it should take a strongly-typed bitflags value.

Reply:
Changed to private, which is sufficient for this PR.

Reply:
I made the function public and make the flags as bitflags vlaue.